tests/binary-expressions/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -154,11 +154,12 @@ const avatar = hasAvatar && <Gravatar user={author} size={size} />;
 
 const avatar = (hasAvatar || showPlaceholder) && <Gravatar user={author} size={size} />;
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-const user = renderedUser || (
-  <div>
-    <User name={this.state.user.name} age={this.state.user.age} />
-  </div>
-);
+const user =
```

Comment:
I think you went too far 😅 Changes like this are regressions...

I think this PR should only affect tagged templates inside a tagged template